illumos-utils/src/running_zone.rs

```diff
@@ -485,7 +492,12 @@ impl RunningZone {
     /// Note that the zone must already be configured to be booted.
     pub async fn boot(zone: InstalledZone) -> Result<Self, BootError> {
         // Boot the zone.
-        info!(zone.log, "Booting {} zone", zone.name);
+        let boot_zone_log = format!("Booting {} zone", zone.name);
+        if zone.name == SWITCH_ZONE_NAME {
```

Comment:
I don't love this conditional:

* it feels awkward
* it doesn't seem right to decide it based on the zone _name_; a "stage" feels like a higher level concept that our caller would attach to the logger if appropriate